rest/src/test/java/org/jboss/pnc/rest/endpoint/BuildRecordEndpointTest.java

```diff
@@ -0,0 +1,50 @@
+package org.jboss.pnc.rest.endpoint;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+import org.jboss.pnc.datastore.repositories.BuildRecordRepository;
+import org.jboss.pnc.model.BuildRecord;
+import org.jboss.pnc.rest.provider.BuildRecordProvider;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ * 
+ * @author Jakub Bartecek <jbartece@redhat.com>
+ *
+ */
+public class BuildRecordEndpointTest {
+
+    @Test
+    public void getLogsNoContentTest() {
+        int logId = 1;
+        BuildRecordEndpoint buildRecordEndpoint = getLogsPrepareEndpoint(logId, "");
+
+        // then
+        assertEquals(204, buildRecordEndpoint.getLogs(logId).getStatus());
+    }
+
+    @Test
+    public void getLogsWithContentTest() {
+        int logId = 1;
+        String logContent = "LOG CONTENT";
+        BuildRecordEndpoint buildRecordEndpoint = getLogsPrepareEndpoint(logId, logContent);
+
+        // then
+        assertEquals(200, buildRecordEndpoint.getLogs(logId).getStatus());
+    }
+
+    private BuildRecordEndpoint getLogsPrepareEndpoint(int logId, String logContent) {
+        // given
```

Comment:
//given //when //then should be added only to the test method's body. This blog post describes it really well: http://monkeyisland.pl/2009/12/07/given-when-then-forever/